src/odemis/gui/cont/acquisition.py

```diff
@@ -1395,10 +1395,15 @@ def update_acquisition_time(self):
             lvl = logging.WARN
         else:
             streams = self._tab_data_model.acquisitionStreams
-            acq_time = acqmng.estimateTime(streams)
-            acq_time = math.ceil(acq_time)  # round a bit pessimistic
-            txt = u"Estimated time is {}."
-            txt = txt.format(units.readable_time(acq_time))
+            folds = tuple(acqmng.foldStreams(self._tab_data_model.acquisitionStreams))
+            if len(streams) > len(folds):
+                txt = u"EBIC and CL streams acquired simultaneously"
```

Comment:
That's true now, but if there are ever different detectors added, this might become incorrect. I think either you detect the name of the streams being folded (which is quite hard) or just keep it generic like "Some streams will be acquired simultaneously".